core/src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java

```diff
@@ -104,7 +109,7 @@ private boolean shouldBeExecuted(Plugin plugin, ProbeContext context) {
      * @param context holder of information passed across the probes executed on a single plugin
      * @return a ProbeResult representing the result of the analysis
      */
-    protected abstract ProbeResult doApply(Plugin plugin, ProbeContext context);
+    protected abstract ProbeResult doApply(Plugin plugin, ProbeContext context) throws IOException;
```

Comment:
Your probe is not even throwing that exception. Why did you made that change?

Reply:
After the third try I was able to fix it without throwing the exception. I added this during the first and second try and forgot to remove later.